form-events-001.md

```diff
@@ -30,17 +30,17 @@ Sounds great, does it ? Sigourney Weaver approves.
 
 What we have forgotten is, our lovely designer friend thought of beautiful animations for the panel to play whenever it’s opening or closing, and bam ! The above flow no longer works. Because we need to unmount the panel component to submit the form data, it will not always been present in the DOM, and so playing animation with it becomes much more complicated.
 
-The key idea here is _the time_. We have to think of intermediate actions that will let us identify when the panel is opening or closing so we can delay the moment it will be unmounted.
+The key idea here is _the time_. We have to think of intermediate actions that will let us identify when the panel is opening or closing, so we can delay the moment it will be unmounted.
 
 Schematically, here is the flow we’re aiming to achieve :
 * a user click on an element to open the pannel. It dispatch an OPEN action,
-* based on the panel’s state, the panel can be mounted in the DOM,
+* based on it’s state, the panel can be mounted in the DOM,
 * a lifecycle hook dispatch a MOUNTED action,
-* the state is updated, we can dynamically append a CSS class to the panel view component, so the sliding animation can be played.
+* the state is updated, we can dynamically append a CSS class to the panel view component, the sliding animation can be played.
 
 The same logic applies when closing the panel, with a subtlety however:
 * a user click on one of the three element that fires the CLOSE action,
```

Comment:
a user **clicks** on one of the three **elements** that **fire** the CLOSE action